client/src/components/search/searchBar/search-bar-optimized.tsx

```diff
@@ -3,7 +3,11 @@ import { useTranslation } from 'react-i18next';
 import Magnifier from '../../../assets/icons/Magnifier';
 import { searchPageUrl } from '../../../utils/algolia-locale-setup';
 
-const SearchBarOptimized = (): JSX.Element => {
+type innerRefPrototype = {
```

Comment:
```suggestion
type Props = {
```
It's more conventional to name a type according to what it _is_, rather than what it _contains_